packages/flutter/lib/src/widgets/bottom_navigation_bar_item.dart

```diff
@@ -24,7 +24,7 @@ class BottomNavigationBarItem {
   /// The argument [icon] should not be null and the argument [title] should not be null when used in a Material Design's [BottomNavigationBar].
   const BottomNavigationBarItem({
     @required this.icon,
-    this.title,
```

Comment:
According to the doc string in the title, it can be null if it is not used in a Material Design [BottomNavigationBar].

Reply:
@chunhtai Thanks